benchmarks/urls.py

```diff
@@ -1,22 +1,40 @@
 from django.urls import path
+import functools
 
 from .views import index, user, model, competition
 
 urlpatterns = [
     # index
-    path('', index, name='index'),
+    path('', functools.partial(index, domain='vision'), name='index'),
+    path('/', functools.partial(index, domain='vision'), name='index'),
     # user
-    path('signup/', user.Signup.as_view(), name='signup'),
-    path('activate/<str:uidb64>/<str:token>', user.Activate.as_view(), name='activate'),
-    path('profile/', user.Profile.as_view(), name='login'),
-    path('logout/', user.Logout.as_view(), name='logout'),
-    path('upload/', user.Upload.as_view(), name='upload'),
-    path('display-name/', user.DisplayName.as_view(), name='display-name'),
-    path('password/', user.Password.as_view(), name='password'),
-    path('password-change/<str:uidb64>/<str:token>', user.ChangePassword.as_view(), name='change-password'),
+    path('logout/', user.Logout.as_view(domain="vision"), name='logout'),
+    path('upload/', user.Profile.as_view(domain="vision"), name='upload'),
     path('public-ajax/', user.PublicAjax.as_view(), name='PublicAjax'),
-    path('resubmit/', user.resubmit, name='resubmit'),
-    # model
-    path('model/<int:id>', model.view, name='model'),
     path('competition/', competition.view, name='competition'),
+
+    # language changes
+    path('vision/', functools.partial(index, domain='vision'), name='index'),
+    path('language/', functools.partial(index, domain='language'), name='index'),
+    path('password/vision/',  user.Password.as_view(domain="vision"), name='password-vision'),
+    path('password/language/',  user.Password.as_view(domain="language"), name='password-language'),
+    path('password-change/vision/<str:uidb64>/<str:token>', user.ChangePassword.as_view(domain="vision"), name='change-password-vision'),
+    path('password-change/language/<str:uidb64>/<str:token>', user.ChangePassword.as_view(domain="language"), name='change-password-language'),
+    path('signup/vision', user.Signup.as_view(domain="vision"), name='vision-signup'),
+    path('signup/language', user.Signup.as_view(domain="language"), name='language-signup'),
```

Comment:
why are there two separate signups? There's just one user table, let's only have `signup`.

Reply:
Will be addressed after new UI landing page (to provide domain - independent redirect)